### PR TITLE
Change allowing to build the website

### PR DESCRIPTION
**Pull request by bam241**
- removed pngmath extensions 
- change reference to `enrichment_mstar_vs_flowrate.ipynb` to `mstar_vs_flowrate.ipynb` and rename files accordingly...

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -26,7 +26,6 @@
 
 extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.doctest',
-              'sphinx.ext.pngmath',
```
Comment (gonuke) [inline docs/conf.py:29]: Is this no longer required for some reason?

**Review by gonuke — COMMENTED** [review]
LGTM - thanks @bam241 - just curious about `pngmath`

File: docs/conf.py (hunk 1 above)
Comment (bam241) [reply docs/conf.py:29]: This prevents the website to build...
This is an issue on some Sphinx versions.. and as it seems to be required...

Comment (bam241) [reply docs/conf.py:29]: https://github.com/sphinx-doc/sphinx/issues/3295

Comment (gonuke) [reply docs/conf.py:29]: try `imgmath`?

Comment (bam241) [reply docs/conf.py:29]: it seems to be working !

Comment (bam241) [reply docs/conf.py:29]: > This prevents the website to build...
> This is an issue on some Sphinx versions.. and as it seems to be required...

*as it seems NOT to be required....

**Comment by gonuke** [comment]
Thanks @bam241

**Pull request merged**